packages/types/src/interfaces/contractsAbi/definitions.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright 2017-2020 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+/* eslint-disable sort-keys */
+
+const layout = {
+  InkLayoutKey: '[u8; 32]',
+  InkStorageLayout: {
+    _enum: {
+      Cell: 'InkLayoutCell',
+      Hash: 'InkLayoutHash',
+      Array: 'InkLayoutArray',
+      Struct: 'InkLayoutStruct',
+      Enum: 'InkLayoutEnum'
+    }
```

Comment:
Nice, got these in. Can you just make the top-level keys alphabetical, since it will create conflicts with the impl. branch as-is. (It needs to be able to be checked against what is there, atm cannot quite do it to evaluate)

Reply:
i.e. `InkLayoutKey` after `InLayoutCell` (and obviously `InkStorageLayout` much later).